LayoutTests/fast/css/cssText-shorthand.html

```diff
@@ -1,24 +1,19 @@
 <!DOCTYPE html>
-<html>
-<body>
-<script src="../../resources/js-test-pre.js"></script>
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
 <script>
-
-description("This test ensures WebKit uses shorthand notations for cssText");
-
 var tests = [
-    // FIXME: This exhibits a bug. We shouldn't be outputing border-image here.
     ['border: 1px; border-top: 1px;', 'border: 1px;'],
     ['border: 1px solid red;', 'border: 1px solid red;'],
     ['border: 1px red;', 'border: 1px red;'],
     ['border: red;', 'border: red;'],
     ['border-top: 1px; border-right: 1px; border-bottom: 1px; border-left: 1px;', 'border: 1px;'],
-    ['border-top: 1px; border-right: 2px; border-bottom: 3px; border-left: 4px;', 'border-width: 1px 2px 3px 4px;'],
-    ['border: 1px; border-top: 2px;', 'border-width: 2px 1px 1px;'],
+    ['border-top: 1px; border-right: 2px; border-bottom: 3px; border-left: 4px;', 'border-color: initial; border-style: initial; border-width: 1px 2px 3px 4px;'],
```

Comment:
I don't think you are really getting `border-color: initial; border-style: initial` without removing this from `asTextInternal()`:

```cpp

        if (propertyID != CSSPropertyCustom && value == "initial"_s && !CSSProperty::isInheritedProperty(propertyID))
            continue;
```